Sources/UIKitExtensions/UITableViewCell+FunctionConfigurationSet.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  UITableViewCellFunctions.swift
+//  ViewConfigurator-iOS
+//
+//  Created by Andrii Pazynych on 27.03.18.
+//  Copyright © 2018 ImagineOn GmbH. All rights reserved.
+//
+
+
+import UIKit
+public extension ConfigurationSet where Base: UITableViewCell {
+    
```

Comment:
all three functions are only useful for existing UITableViewCells whose isEditing, isSelected and isHighlighted change should be animated.
The  isEditing, isSelected and isHighlighted Properties are writable directly so there is no need for this function.